### PR TITLE
Open adjacent tabs and background tabs

### LINKED ISSUE #60: Option to open tabs from follow in the background
### I'm opening this issue because:

- [x] I'll propose a new feature

### Description

It'd be nice to be able to have new tabs opened from follow hints to be done in the background.

### PR DESCRIPTION
**Pull request by idlewan**
Thanks to #62, I updated the "open in background tab" pull request, and added the possibility to open tabs right next to the current one instead of at the end of the tabs list.

Closes #60 and #62.

How does it look ?

### CODE REVIEW

**Comment by ueokande** [comment]
@idlewan please resolve conflicts

File: src/shared/default-settings.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ export default {
       "twitter": "https://twitter.com/search?q={}",
       "wikipedia": "https://en.wikipedia.org/w/index.php?search={}"
     }
-  }
+  },
+  "openAdjacentTabs": false
```
Comment (ueokande) [inline src/shared/default-settings.js]: please use properties (refer #306 as an example)

Comment (ueokande) [reply src/shared/default-settings.js]: I think short property name like `adjacenttabs` is better

**Comment by mathstuf** [comment]
@idlewan Any progress here?

**Comment by idlewan** [comment]
Haven't looked at merging yet because life. Will probably at some point, but might be in weeks. Feel free to take over and work on it if you want @mathstuf

**Comment by mathstuf** [comment]
OK. I may take it up if I get the time, but I was just making sure this wasn't just lost in a TODO list somewhere :) .

**Comment by ueokande** [comment]
@idlewan 
Thank you for your pull request.  

Adjacent tabs are now configured by `"adjacenttab"` property (default to `true`).  To configure default behavior, add a property like the following:

```json
{
   "properties": {
    "adjacenttab": false
  }
}
```

Use can configure it ephemerally by `:set` command:

```
:set adjacenttab
:set noadjacenttab
```

**Pull request merged**